src/components/TriggerFolderLink.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { Component, PureComponent } from 'react'
+import classNames from 'classnames'
+
+import Icon from 'cozy-ui/react/Icon'
+
+import styles from '../styles/triggerFolderLink'
+
+const ERRORED = 'ERRORED'
+const FETCHING = 'FETCHING'
+const SUCCESS = 'SUCCESS'
+
+/**
+ * Cache
+ */
+let driveUrl
```

Comment:
I'm not very comfortable with variables floating around  like that. It does work but they live outside of react and are use inside it, and I always have a hard time figuring out when exactly they are imported, initialized, etc. But maybe that's just me.

I think a better place to store this would be in the app's global state (in redux, or the context). But I think that's more or less where it was, so curious to know why that changed?

Reply:
The role of this variable is purely to cache the information.

Only this component need this info, so I find it overkill to store it all along the whole application.

Also, we aim to use intent redirection to Drive instead of this kind of hack too much sensible to any change in Drive urls. But it needs a redirection mechanism in Drive (as we implemented into the Store). When we will implement it, we will only change the implementation on this component without any other effect in the rest of the application.

Ideally, cozy-client should cache the apps request result, but it is not doing it yet.

BUT maybe it could fit better into the redux store. Let me give it a try.

Reply:
Ok you we right, it is much better now with drive apps properties stored in redux store, thanks !